stronghold/stronghold.gd

```diff
@@ -1,44 +1,82 @@
 class_name Stronghold
 extends Node2D
 
+@onready var occupants: Node = $Occupants
+@onready var sprites: Node2D = $Sprites
+
 var _health: Health
+var _level: int = 0
+var _max_level: int = 2
```

Comment:
Just wondering if the level and max level keeping would be better suited in the upgrade itself? It could make a few things simpler.

Reply:
yeah it'd be better on the upgrade